cmd/crates/soroban-spec-typescript/src/project_template/src/index.ts

```diff
@@ -73,21 +72,20 @@ if (typeof window !== 'undefined') {
     window.Buffer = window.Buffer || Buffer;
 }
 
-const regex = /ContractError\((\d+)\)/;
+const regex = /Error\(Contract, #(\d+)\)/;
 
-function getError(err: string): Err<Error_> | undefined {
-    const match = err.match(regex);
+function parseError(message: string): Err<Error_> | undefined {
+    const match = message.match(regex);
     if (!match) {
         return undefined;
     }
-    if (Errors == undefined) {
+    if (Errors === undefined) {
```

Comment:
this confused me initially and does still: where is `Errors` supposed to be defined?

Reply:
It is injected by the rust. Basically if the contract has errors defined they are used otherwise it's initialized as undefined.